.github/workflows/codeql-analysis.yml

```diff
@@ -39,10 +38,10 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v2
-    - name: Set up JDK 1.8
+    - name: Set up JDK 1.9
```

Comment:
Is this the only repo where we use Java 9?

Reply:
see my other comment